plugins/plugins/__init__.py

```diff
@@ -1651,8 +1649,8 @@ def _operator_skeleton_placement_code(ctx):
         def resolve_placement(self, ctx):
             return types.Placement(
                 types.Places.{placement},
-                label = "{placement_label}"
-                icon = {placement_icon}
+                label = "{placement_label}",
```

Comment:
nit: can you also remove the whitespace around the `=`? IE `label=value` rather than `label = value`

Reply:
Yep! will do right now :)